docs/self-managed/deployment/helm/configure/authentication-and-authorization/external-idp-via-internal-keycloak.md

```diff
@@ -0,0 +1,163 @@
+
+#### Grant component access
+
+Grant access to Orchestration Cluster components:
+
+1. Navigate to **Authorizations** > select **Component** > **Create authorization**.
+1. Configure the authorization:
+   - **Owner type**: `Mapping rule`
+   - **Owner ID**: `external-idp-users-rule`
+   - **Resource ID**: `*`
+   - **Permissions**: `ACCESS`
+
+#### Grant additional permissions (optional)
+
+Grant additional permissions as needed. For example, to allow users to view processes and complete tasks:
+
+| Resource Type      | Resource ID | Permissions                                                          |
+| ------------------ | ----------- | -------------------------------------------------------------------- |
+| Process Definition | `*`         | `READ_PROCESS_DEFINITION`, `READ_PROCESS_INSTANCE`, `READ_USER_TASK` |
+| Process Definition | `*`         | `UPDATE_USER_TASK`                                                   |
+
+:::info
+For more details on authorizations, see [Orchestration Cluster authorization](/components/concepts/access-control/authorizations.md).
+:::
+
+### Configure Management Identity access (Optional)
+
+For access to Console, Web Modeler, and Optimize, external IdP users need the corresponding realm roles assigned in Keycloak. The recommended approach is to assign users to groups that have these roles.
+
+#### Verify or create groups
+
+1. In Keycloak Admin Console, navigate to **Groups**.
+1. Verify that groups exist for each component (e.g., `Console`, `Optimize`, `Web Modeler`). If not, create them.
+
+#### Assign roles to groups
+
+Ensure each group has the corresponding realm role assigned:
+
+1. Select the group > **Role Mappings** tab.
+1. Click **Assign role** and add the role with the same name (e.g., `Console`).
+
+#### Create group mappers
+
+Create mappers to assign federated users to these groups:
+
+1. Navigate to **Identity Providers** > select your IdP > **Mappers** tab.
+1. Click **Add mapper** for each component:
+
+| Mapper Name               | Mapper Type     | Group         |
+| ------------------------- | --------------- | ------------- |
+| `assign-console-group`    | Hardcoded Group | `Console`     |
+| `assign-optimize-group`   | Hardcoded Group | `Optimize`    |
+| `assign-webmodeler-group` | Hardcoded Group | `Web Modeler` |
+
+:::tip
+You can also assign roles directly to users in Keycloak, or use [mapping rules in Management Identity](/self-managed/components/management-identity/mapping-rules.md) to map token claims to roles.
+:::
+
+## Understanding the two identity systems
+
+Camunda 8.8 uses two separate identity systems: Orchestration Cluster Identity and Management Identity. Each system manages access to different components and uses different configuration methods.
+
+For a detailed explanation of the differences between these identity systems, see [Identity types in Camunda 8](/components/concepts/access-control/access-control-overview.md#identity-types-in-camunda-8).
```

Comment:
I wonder if we should have this section here in this specific guide, because it applies generally to all deployment setups that use both OC and Management components.